src/module/macros.ts

```diff
@@ -1324,23 +1298,23 @@ export async function prepareStructureMacro(a: string) {
 
 export async function prepareActivationMacro(a: string, i: string, type: ActivationOptions, index: number) {
   // Determine which Actor to speak as
-  let actor: Actor | null = getMacroSpeaker(a);
+  let actor = getMacroSpeaker(a);
   if (!actor) return;
```

Comment:
Changing this to `if (!actor || !actor.is_mech()) return;` eliminates the need for the `ts-ignore` on line 1317. (For now, until we add activation macros to NPCs...)